Source/NSURLConnection.m

```diff
@@ -157,13 +157,37 @@ + (NSURLConnection *) connectionWithRequest: (NSURLRequest *)request
   return AUTORELEASE(o);
 }
 
+- (void) start
+{
+  [this->_protocol startLoading];
+}
+
 - (void) cancel
 {
   [this->_protocol stopLoading];
   DESTROY(this->_protocol);
   DESTROY(this->_delegate);
 }
 
+- (void) scheduleInRunLoop: (NSRunLoop *)aRunLoop 
+                   forMode: (NSRunLoopMode)mode
+{
+  NSArray *modes = [NSArray arrayWithObject: mode];
+  [aRunLoop performSelector: @selector(start)
+                     target: self
+                   argument: nil
+                      order: 0
+                      modes: modes];
+}
+
+- (void) unscheduleFromRunLoop: (NSRunLoop *)aRunLoop 
+                       forMode: (NSRunLoopMode)mode
```

Comment:
It looks wrong that the mode argument gets ignored, but I see no way of using it.

Reply:
That was my confusion as well.